src/app/pages/HomePage/HomePage.tsx

```diff
@@ -33,6 +33,8 @@ export interface HomePageProps {
     title: string;
     curations: Curation[];
     description: string;
+    seoTitle: string | undefined;
+    seoDescription: string | undefined;
```

Comment:
```suggestion
    seoTitle?: string;
    seoDescription?: string;
```
Minor syntax suggestion not seen an optional parameter expressed like this before?